Source/JavaScriptCore/runtime/JSObject.cpp

```diff
@@ -3196,6 +3196,79 @@ bool JSObject::putByIndexBeyondVectorLengthWithArrayStorage(JSGlobalObject* glob
     return true;
 }
 
+void JSObject::copyArrayStorage(JSGlobalObject* globalObject, JSObject* other)
```

Comment:
Contiguous etc., is not `ArrayStorage`, so this name looks wrong.

Reply:
Removed.